server/src/resolvers/getVariantsResolver/adapters/g4rdAdapter.ts

```diff
@@ -68,7 +71,7 @@ const getG4rdNodeQuery = async ({
   const url = `${process.env.G4RD_URL}/rest/variants/match`;
   const { position, ...gene } = geneInput;
   try {
-    G4RDNodeQueryResponse = await axios.post<G4RDQueryResult>(
+    G4RDVariantQueryResponse = await axios.post<G4RDVariantQueryResult>(
```

Comment:
I don't have much too say on this, but maybe keep local variables camelCase to conform to convention and avoid confusion with types, interfaces, and classes.